enhancements/olm-toggle-copied-csvs.md

```diff
@@ -0,0 +1,199 @@
+
+When toggled (on then off), OLM will recreate any missing Copied CSVs and reassociate the owner references of related copied resources with them (i.e. the inverse operation of the one described in the [Copied RBAC](#copied-rbac) section).
+
+The toggle will be disabled by default.
+
+The toggle can be set from a command-line option on the olm-operator binary.
+
+#### Config CRD w/ Toggle
+
+A novel cluster-scoped `OLMConfig` CRD will be added to OLM. The resource type it defines will allow users to apply a limited set of configurations to the OLM instance on their cluster by creating or modifying a singlton (of that type) on the cluster.
+
+The name of that singleton will be `config` and OLM will ignore all other `OLMConfig` resources on a cluster.
+
+Its spec will contain a Copied CSV feature toggle field.
+
+e.g.
+
+```yaml
+apiVersion: operators.coreos.com/v1
+kind: OLMConfig
+metadata:
+  name: config
+spec:
+  features:
+    'disable-copied-csvs': true
+```
+
+When both the [command-line option](#the-toggle) _and_ `config` toggle field are present, the latter takes precedence.
+
+When the `config` resource isn't present, OLM uses its default settings.
+
+### Risks and Mitigations
+
+#### Loss of Operator Discovery
+
+**Risks:** When Copied CSVs are disabled, users won't be able to discover what operators are configured to watch a particular namespace unless they have access to their install namespaces.
+
+**Potential Mitigations:** 
+
+- Reduce number of impacted users by limiting toggle to operators installed in `AllNamespace` mode only; that is, `Single` and `MultiNamespace` mode Copied CSVs will not be affected by the toggle.
+- OLM fires an alert in all affected namespaces on startup when Copied CSVs are disabled
+- Docs suggest admins notify their cluster tenants of installed operators directly; i.e. email, slack, etc
+
+#### OpenShift Console Integration
+
+**Risk:** The OpenShift Console uses Copied CSVs in the rendering of certain pages, so disabling Copied CSVs will negatively impact UX.
+
+For the **admin console**, Copied CSVs are used to "drill down" on the details of operators watching a particular namespace:
+
+"Installed Operators" page (i.e. a list of Copied CSVs) 
+└── "Operator Details" page (e.g. Copied CSV of Strimzi)
+            └── "Operand List" page (i.e. instance list of provided APIs through Copied CSV)
+                                    └── "Operand Details" page (e.g. kafka instance details through Copied CSV)
+
+For the **dev console**, Copied CSVs drive the creation and tracking of CRs.
+
+**Potential Mitigations:**
+
+- Console detects when Copied CSVs are diabled and displays a warning to admins and affected users
+- A toggle component is added to Console to make re-enabling Copied CSVs easier
```

Comment:
same here....is this part of the EP, or not?  (i think it should be, w/ console team sign off of course)